### ISSUE TITLE
URL of wireshark's server for version 3.4.9 has changed

### ISSUE THREAD

**Issue by GeorgeTasop** [open-quantum-safe/oqs-demos#112]
This issue is for the Wireshark demo:

While trying to use the Wireshark demo, I run the `build.sh` script only to find that the path on Wireshark's download server (at line 43) didn't find anything. The script declares ${WIRESHARK_VERSION}=3.4.9, so the correct URL should be the following:

`https://2.na.dl.wireshark.org/src/all-versions/wireshark-${WIRESHARK_VERSION}.tar.xz`

Probably Wireshark version 3.4.9 has moved from /src/ directory in the directory /src/all-versions/ in Wireshark's download server.

**Comment by GeorgeTasop** [comment]
As a matter of fact, I think this affects the current Dockerfile too, because I get the same error when I run the docker command specified at "Quickstart": 

`docker build -t openquantumsafe/wireshark .`

 The output from Step 10/20 is the following:

```
Step 10/20 : RUN curl --output wireshark-${WIRESHARK_VERSION}.tar.xz https://2.na.dl.wireshark.org/src/wireshark-${WIRESHARK_VERSION}.tar.xz && tar xmvf wireshark-${WIRESHARK_VERSION}.tar.xz
 ---> Running in b1275200db27
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1556    0  1556    0     0   1612      0 --:--:-- --:--:-- --:--:--  1610
xz: (stdin): File format not recognized
tar: Child returned status 1
tar: Error is not recoverable: exiting now
The command '/bin/sh -c curl --output wireshark-${WIRESHARK_VERSION}.tar.xz https://2.na.dl.wireshark.org/src/wireshark-${WIRESHARK_VERSION}.tar.xz && tar xmvf wireshark-${WIRESHARK_VERSION}.tar.xz' returned a non-zero code: 2
```

I think the errors from the `tar` command arises because the respective tar.xz file was not found in the URL and returned an empty file.

**Comment by baentsch** [comment]
Thanks for your interest in our work and this clear bug report. Working on it.

**Comment by baentsch** [comment]
Closed via #113. Thanks again for letting us know, @GeorgeTasop

**Comment by GeorgeTasop** [comment]
You are welcome :)
